packages/aragon-cli/src/commands/apm_cmds/publish.js

```diff
@@ -713,9 +723,8 @@ exports.handler = async function({
   )
 
   if (contentProvier === 'ipfs') {
+    reporter.info('Explore IPFS content locally with the following link:')
     console.log(
-      '\n',
-      'Explore the ipfs content locally:',
       '\n',
       chalk.bold(
         `http://localhost:8080/ipfs/QmSDgpiHco5yXdyVTfhKxr3aiJ82ynz8V14QcGKicM3rVh/#/explore/${contentLocation}`
```

Comment:
Having both of these hashes is a little bit confusing; not sure if we can do a great job explaining this, but it would be nice if users could easily understand that this first hash is the IPFS explorer's frontend.